<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgSvgTextSettings</class>
 <widget class="QWidget" name="WdgSvgTextSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>331</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Keyword</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorKeyword"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkBoldKeyword">
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkItalicKeyword">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Element</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorElement"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkBoldElement">
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkItalicElement">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Attribute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorAttribute"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkBoldAttribute">
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkItalicAttribute">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string comment="The Value label used by SVG Text Tool">Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorValue"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkBoldValue">
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkItalicValue">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="colorComment_2">
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="colorComment"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkBoldComment">
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkItalicComment">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Editor text color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KColorButton" name="colorEditorForeground"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Editor background:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KColorButton" name="colorEditorBackground"/>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
